src/components/app/details/cicdHistory/TriggerDetails.tsx

```diff
@@ -87,17 +90,25 @@ export const TriggerDetails = React.memo(
     },
 )
 
-const Finished = React.memo(({ status, finishedOn, artifact }: FinishedType): JSX.Element => {
+const Finished = React.memo(({ status, startedOn, finishedOn, artifact }: FinishedType): JSX.Element => {
     return (
         <div className="flex column left">
             <div className={`${status} fs-14 fw-6 ${TERMINAL_STATUS_COLOR_CLASS_MAP[status.toLowerCase()] || 'cn-5'}`}>
                 {status && status.toLowerCase() === 'cancelled' ? 'ABORTED' : status}
             </div>
+            
             <div className="flex left">
                 {finishedOn && finishedOn !== ZERO_TIME_STRING && (
-                    <time className="cn-7 fs-12 mr-12">
-                        {moment(finishedOn, 'YYYY-MM-DDTHH:mm:ssZ').format(Moment12HourFormat)}
-                    </time>
+                    <>            
+                        <time className="dc__vertical-align-middle">
+                            {moment(finishedOn, 'YYYY-MM-DDTHH:mm:ssZ').format(Moment12HourFormat)}
+                        </time>
+                        <div className="dc__bullet mr-6 ml-6"></div>
```

Comment:
Can use <div className="dc__bullet mr-6 ml-6"/>  if no children.

Reply:
Ok sure, Done that